### PR TITLE
Question card

### PR DESCRIPTION
**Pull request by halimahexe**
I've created the UI for the question card, setting up new CSS classes for each of the parts. I think these could be made reusable but I wanted to make sure it all looked right.

I've used a `border-radius: 8px` throughout. I've played around with padding a little bit just because some elements looked too big with the standard `16px 24px`.

Ideally I wanted the `.btn-ternary` (edit and fave buttons) to change fill colour on hover, utilising the SVG functionality but I couldn't get it to work so decided to just change the background on hover for simplicity.

The `.icons` have a width/height of 24px.

You'll see I've put variable names which have been commented out, while I've styled everything. Eventually, this should connect to a function that fetches random questions and passes into the component. I've also created a new type `QuestionCardProps` which I think should actually just be replaced with the original  `Question` type we created, as it's exactly the same thing. That's in the backend though, so I wasn't sure how to connect it.

### CODE REVIEW

**Review by oskarprzybylski23 — COMMENTED** [review]
I think it would be good to import useEffect and useContext to avoid using React.method to keep it consistent with other components

**Comment by halimahexe** [comment]
> I think it would be good to import useEffect and useContext to avoid using React.method to keep it consistent with other components

Thanks Oskar - I refactored so I wasn't using `React.method` before pushing. Please could you have another look to make sure I haven't missed anything and point out specific places if you see them?

**Review by oskarprzybylski23 — APPROVED** [review]
LGTM

File: src/App.tsx [hunk 1]
```diff
@@ -19,9 +20,7 @@ function App() {
 					btnText='dolo'
 					btnonClick={() => false}
 				/>
-				<div>
-					<QuestionCard />
-				</div>
+				<div>{<QuestionCard />}</div>
```
[the remark this answers is not in the document]
Comment (halimahexe) [reply src/App.tsx:23]: I've fixed that in #29

**Pull request closed without merging**